packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -408,7 +404,9 @@ function TabPicker<T>(props: TabPickerProps<T>) {
           'spectrum-Tabs--emphasized': isEmphasized
         },
         className
-      )}>
+      )}
+      style={style}
+      aria-hidden={!visible}>
```

Comment:
I think aria-hidden is one of those fields, where, instead of false, it should be omitted
```suggestion
      aria-hidden={visible ? undefined : true}>
```